CRM/Contact/Form/Relationship.php

```diff
@@ -578,22 +578,23 @@ private function createAction($params) {
   /**
    * Prepares parameters to be used for create/update actions
    *
-   * @param array $params
+   * @param array $values
    *
    * @return array
    */
-  private function preparePostProcessParameters($params) {
-    $relationshipTypeParts = explode('_', $params['relationship_type_id']);
+  private function preparePostProcessParameters($values) {
+    $params = $values;
+    list($relationshipTypeId, $relationshipContactA, $relationshipContactB) = explode('_', $params['relationship_type_id']);
```

Comment:
Just cleanup to make it easier to read this function so we don't rely on numeric array IDs.